flagging_site/data/database.py

```diff
@@ -33,6 +33,34 @@ def execute_sql_from_file(file_name: str):
         return execute_sql(f.read().decode('utf8'))
 
 
+def create_db():
+    """If database doesn't exist, create it and return True, 
+    otherwise leave it alone and return False"""
+
+    # connect to postgres database, get cursor
+    conn = connect(dbname='postgres', user='postgres', \
+        host=current_app.config['POSTGRES_HOST'], \
+            password=current_app.config['POSTGRES_PASSWORD'])
+    cursor = conn.cursor()
```

Comment:
I got it working on my computer (Macbook) with just this one change:

```python
    conn = connect(
        dbname='postgres',
        user=current_app.config['POSTGRES_USER'],
        host=current_app.config['POSTGRES_HOST'],
        password=current_app.config['POSTGRES_PASSWORD']
    )
```

Basically I just parametrized the `user` so that it is using the `current_app.config['POSTGRES_USER']`. From there, I store in my `.env` the value `POSTGRES_USER=my_computer_username`.

Reply:
One more note about this change I made. You'll notice I'm not using `\`. That's because Python doesn't require explicit line continuations for open parentheses or brackets. Because editing code becomes more cumbersome when you have to manage the line continuations, it's preferable to avoid them in situations where you can continue lines without them.

Reply:
I have committed new code today which addressed your concerns expressed above, as well as other communications we've had since then